be/src/exec/json_scanner.cpp

```diff
@@ -598,9 +601,11 @@ Status JsonReader::_construct_row_without_jsonpath(simdjson::ondemand::object* r
             if (UNLIKELY(i == _op_col_index)) {
                 // special treatment for __op column, fill default value '0' rather than null
                 if (column->is_binary()) {
-                    std::ignore = column->append_strings(std::vector{Slice{"0"}});
+                    std::string op = is_delete ? "1" : "0";
```

Comment:
It's recommend to extract "1", "0" into a PrimaryKeyOps class.

Reply:
> It's recommend to extract "1", "0" into a PrimaryKeyOps class.

Ok